### PR TITLE
fix: [DHIS2-10702] - Unexpected error querying tracked entity instance porting  [2.34]

### PR DESCRIPTION
**Pull request by lucaCambi77**
see https://github.com/dhis2/dhis2-core/pull/7722

### CODE REVIEW

File: dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityType.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2004-2021, University of Oslo
+ * Copyright (c) 2004-2019, University of Oslo
```
Comment (gnespolino) [inline dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityType.java:2]: should be 2021

File: dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/metadata/objectbundle/hooks/TrackedEntityTypeObjectBundleHookTest.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2004-2019, University of Oslo
```
Comment (gnespolino) [inline dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/metadata/objectbundle/hooks/TrackedEntityTypeObjectBundleHookTest.java:2]: should be 2021

File: dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/metadata/objectbundle/validation/TrackedEntityTypeValidationTest.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2004-2020, University of Oslo
```
Comment (gnespolino) [inline dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/metadata/objectbundle/validation/TrackedEntityTypeValidationTest.java:2]: same here

**Review by gnespolino — APPROVED** [review]

File: dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.33/V2_33_36__Add_translations_column_into_programruleaction_table.sql [hunk 4]
```diff
@@ -0,0 +1 @@
+alter table programruleaction add column if not exists translations jsonb;
```
Comment (stian-sandvold) [inline dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.33/V2_33_36__Add_translations_column_into_programruleaction_table.sql:1]: This is a script for 2.33, and not 2.34, so it should be fixed on the 2.33 branch, not 2.34. Or is this a relevant problem for 2.34?

I would in either case suggest removing this file from this PR, and fixing it in a separate PR

**Review by stian-sandvold — CHANGES_REQUESTED** [review]
See comment about flyway script for 2.33

**Review by stian-sandvold — APPROVED** [review]

**Pull request merged**